compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/utils.kt

```diff
@@ -226,6 +227,21 @@ fun PsiElement.toLocation(): Location {
     return FileLocation(file.virtualFile.path, document.getLineNumber(this.textOffset) + 1)
 }
 
+private fun parseDocString(raw: String): String? {
+    val t1 = raw.trim()
+    if (!t1.startsWith("/**") || !t1.endsWith("*/"))
+        return null
+    val lineSep = t1.findAnyOf(listOf("\r\n", "\n", "\r"))?.second ?: ""
+    return t1.trim('/').trim('*').lines().joinToString(lineSep) {
+        it.trimStart().trimStart('*')
+    }
+}
+
+fun PsiElement.getDocString(): String? =
+    this.firstChild.siblings().firstOrNull { it is PsiComment }?.let {
```

Comment:
Should we take care of the case when normal comments are used with document comments?

Reply:
What do you mean by normal comments? `//`, `/* */` or ...?

Reply:
both

Reply:
No, we should keep it simple and not include random comments.

Reply:
Consider this code:
```
// comment 1
/**
* document 
*/ 
class Foo
```
what will happen with your implementation?

Reply:
comment 1 will be ignored.